examples/example-jwt-auth/README.md

```diff
@@ -0,0 +1,65 @@
+Authentication Example
+==============================================
+
+This example illustrates a simple JWT-based authentication implementation in gRPC using
+ server interceptor. It uses the JJWT library to create and verify JSON Web Tokens (JWTs).
+
+The example requires grpc-java to be pre-built. Using a release tag will download the relevant binaries
+from a maven repository. But if you need the latest SNAPSHOT binaries you will need to follow
+[COMPILING](../../COMPILING.md) to build these.
+
+The source code is [here](src/main/java/io/grpc/examples/jwtauth). Please follow the
+[steps](../README.md#-to-build-the-examples) to build the examples. The build creates scripts
+`auth-server` and `auth-client` in the `build/install/examples/bin/` directory which can be
+used to run this example. The example requires the server to be running before starting the
+client.
+
+Running auth-server is similar to the normal hello world example and there are no arguments to supply:
+
+**auth-server**:
+
+```text
+USAGE: AuthServer
```

Comment:
What's the value of this text here? User almost never invokes AuthServer class with java command and the current AuthServer class does not have USAGE output like above. And the command does not take any arguments.

Even if this format/template was copied from another example we can clean up the content as and when needed.

Reply:
This is not for the AuthServer class, but for the generated auth-server script. I will fix the command.